app/src/main/java/ch/epfl/skysync/viewmodel/CalendarViewModel.kt

```diff
@@ -31,31 +29,32 @@ data class CalendarUiState(
  * ViewModel for the Calendar screen
  *
  * @param uid The Firebase authentication uid of the user
- * @param userTable The user table
- * @param availabilityTable The availability table
+ * @param repository App repository
  */
 class CalendarViewModel(
     private val uid: String,
```

Comment:
Is there an advantage to passing the user id rather than the object User ?

Reply:
That will be possible once #263 is done

Reply:
Perfect